src/main/kotlin/no/nav/familie/ef/iverksett/domene/Inntekt.kt

```diff
@@ -1,13 +1,11 @@
 package no.nav.familie.ef.iverksett.domene
 
-import no.nav.familie.ef.iverksett.infrastruktur.json.PeriodebeløpJson
-
 data class Inntekt(
-    val periodeBeløp: PeriodeBeløp,
+    val periodebeløp: Periodebeløp,
```

Comment:
Vi spør ikke om inntektstype i saksbehandlingsløsningen - er dette noe vi tenker er påkrevd for videre "prosess" ? Må isåfall tas opp med Lars?

Reply:
@joakibj

Reply:
Lar denne ligge foreløbig, og noterer ned dette